core/src/main/java/io/confluent/connect/storage/StorageSinkTestBase.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 Confluent Inc.
```

Comment:
Please don't change copyright dates of existing files.

Reply:
Ah, okay. I was under the impression that touching a file meant to dump the date.